src/js/components/AccordionPanel/doc.js

```diff
@@ -21,6 +21,16 @@ export const themeDoc = {
     type: 'number',
     defaultValue: '4',
   },
+  'accordion.heading.margin': {
+    description: 'The margin around heading in the accordion.',
+    type: 'string',
+    defaultValue: 'undefined',
+  },
+  'accordion.heading.pad': {
```

Comment:
Semantically, the pad isn't assigned to heading, so I think it should be accordion.pad

Reply:
I saw that as it was on `box` Ill change thanks

Reply:
what about fixing it on the docs as well?